apollo-runtime/src/main/java/com/apollostack/android/GraphqlService.java

```diff
@@ -0,0 +1,14 @@
+package com.apollostack.android;
+
+import android.database.Observable;
+
+import java.util.List;
+
+import javax.annotation.Nullable;
+
+/** A {@link retrofit2.Retrofit} service for executing GraphQL requests. */
+public interface GraphqlService {
```

Comment:
I would rather generate this interface or maybe even it won't be required to be dynamic (generated) but rather compile time (will ship with runtime module) as I see it's used only internally. So from user perspective the API should be very simple: `Apollo.execute(query, callback)` or with Rx `Apollo.execute(query).subscribe()` and he shouldn't care how this execution was implemented (using or not retrofit, etc. ). Yes if user decides to set-up own converter we should allow that but that will expose only settings of `Retrofit.Builder` not service.